lib/web_ui/lib/src/engine/canvaskit/fonts.dart

```diff
@@ -11,13 +11,32 @@ import '../util.dart';
 import 'canvaskit_api.dart';
 import 'font_fallbacks.dart';
 
-// This URL was found by using the Google Fonts Developer API to find the URL
-// for Roboto. The API warns that this URL is not stable. In order to update
-// this, list out all of the fonts and find the URL for the regular
-// Roboto font. The API reference is here:
-// https://developers.google.com/fonts/docs/developer_api
-const String _robotoUrl =
-    'https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Me5WZLCzYlKw.ttf';
+// Runtime parameters.
+//
+/// This URL was found by using the Google Fonts Developer API to find the URL
+/// for Roboto. The API warns that this URL is not stable. In order to update
+/// this, list out all of the fonts and find the URL for the regular
+/// Roboto font. The API reference is here:
+/// https://developers.google.com/fonts/docs/developer_api
```

Comment:
actually this paragraph should be a regular comment (`//`) not api docs (`///`) because it's documenting how to update the code, not how to use the constant.

Reply:
done.